packages/roosterjs-editor-plugins/lib/ContentEdit/features/shortcutFeatures.ts

```diff
@@ -64,7 +64,9 @@ function cacheGetCommand(event: PluginKeyboardEvent) {
     return cacheGetEventData(event, 'DEFAULT_SHORT_COMMAND', () => {
         let e = event.rawEvent;
         let key =
-            event.eventType == PluginEventType.KeyDown
+            // Need to check ALT key to be false since in some language (e.g. Polski) uses AltGr to input some special charactors
+            // In that case, ctrlKey and altKey are both true in Edge, but we should not trigger any shortcut function here
+            event.eventType == PluginEventType.KeyDown && !e.altKey
```

Comment:
As aluded to in the comment, ctrlKey and altKey are true for AltGraph being pressed. Want to make sure we want to explicitly check this case? Or would we prefer to do a check like what I've got here. Is AltGraph part of any shortcuts in other languages? I know Alt + key is sometimes used which is why I'm hesitant.
```suggestion
            event.eventType == PluginEventType.KeyDown && e.key != AltGraph
```

Reply:
Just remembered that when spoofing the keyboard, I'm not sure edge recognizes e.key as AltGraph. So maybe this change isn't warranted.

Reply:
At this point e.key is letter key user pressed. In the case I fixed, it is "z", so (e.key != altGr) is always true here.